salt/utils/sdb.py

```diff
@@ -63,3 +63,31 @@ def sdb_set(uri, value, opts):
 
     loaded_db = salt.loader.sdb(opts, fun)
     return loaded_db[fun](query, value, profile=profile)
+
+
+def sdb_delete(uri, opts):
+    '''
+    Delete a value from a db, using a uri in the form of ``sdb://<profile>/<key>``. If
+    the uri provided does not start with ``sdb://`` or the value is not successfully
+    deleted, return ``False``.
+    '''
+    if not isinstance(uri, string_types):
```

Comment:
Good catch, thanks! Fixed.

Actually, I copied the behavior from `sdb.set`. Looking at it right now I see that it has the same issue. I can fix it as part of the same PR or submit another one. :)

P.S.: Guess we need some kind of dummy in-memory `sdb` driver that we can use for easy testing so we can catch such things with unit tests.